### PR TITLE
chore(styled-components): deprecate extend

### PR DESCRIPTION
**Pull request by brad-decker**
Breaking Change:
* Recommended API for adding styles to components is different, no extending.

### CODE REVIEW

**Comment by dekkofilms** [comment]
Just want to make sure that the flex-grid page is okay ... it seems like in the deployed link nothing is showing up. Otherwise looks great!

**Comment by brad-decker** [comment]
Thanks for catching that @dekkofilms will check this out!

File: src/icons/svg-icons/AcUnit.js [hunk 1]
```diff
@@ -1,13 +1,14 @@
...
 
 const AcUnitPath = () => [
   <path d="M22 11h-4.17l3.24-3.24-1.41-1.42L15 11h-2V9l4.66-4.66-1.42-1.41L13 6.17V2h-2v4.17L7.76 2.93 6.34 4.34 11 9v2H9L4.34 6.34 2.93 7.76 6.17 11H2v2h4.17l-3.24 3.24 1.41 1.42L9 13h2v2l-4.66 4.66 1.42 1.41L11 17.83V22h2v-4.17l3.24 3.24 1.42-1.41L13 15v-2h2l4.66 4.66 1.41-1.42L17.83 13H22z" key='path0' />,
 ];
 
-const AcUnitIcon = Icon.extend.attrs({
+const AcUnitIcon = styled(Icon).attrs({
   children: AcUnitPath,
-})``;
+});
```
Comment (petegivens) [inline src/icons/svg-icons/AcUnit.js]: I believe this needs the backticks to invoke the function, even if we aren't applying any styles.

**Comment by brad-decker** [comment]
This is FINALLY COMPLETED IN #325

**Pull request closed without merging**